packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 * Fixes violations of new analysis option use_named_constants.
 * Removes an unnecessary override in example code.
 
+## 2.3.10
+
+* Added option to configure the User-Agent in the HTTP headers for the ExoPlayer agent.
```

Comment:
Same style issue here. Also, this was merged incorrectly at some point, as it's adding a duplicate of an existing version now.